js/packages/teams-ai/src/internals/types.ts

```diff
@@ -82,8 +22,7 @@ export interface CreateCompletionResponseUsage {
  */
 export interface CreateChatCompletionRequest {
     messages: Array<ChatCompletionRequestMessage>;
-    functions?: Array<ChatCompletionAction>;
-    function_call?: CreateChatCompletionRequestFunctionCall;
+    model: string;
     temperature?: number | null;
     top_p?: number | null;
```

Comment:
Could these be alphabetized (except when differentiating between required and optional)?